auth-server/README.md

```diff
@@ -29,6 +29,8 @@ To deploy the [`Auth server`](/auth-server/) manually:
 1. [Create](https://cloud.google.com/compute/docs/instances/create-start-instance) a Compute Engine VM instance with your preferred machine type and OS (for example, e2-medium and Debian 10), then [reserve a static IP](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-internal-ip-address)
 1. Verify that your VM instance has the `Stackdriver Logging API` write [access scope](https://cloud.google.com/compute/docs/access/service-accounts#accesscopesiam) (on by default) and that your VM’s [service account](https://cloud.google.com/compute/docs/access/service-accounts#default_service_account) has the [`Logs Writer`](https://cloud.google.com/logging/docs/access-control) role (off by default) 
 1. Check out the latest code from the [FDA MyStudies repository](https://github.com/GoogleCloudPlatform/fda-mystudies/)
+1. Update the organization name as desired for org.name in the [application.properties](oauth-scim-service/src/main/resources/application.properties) file
+1. Update the landing page logo as desired replacing the image [landingpage-logo.png](oauth-scim-service/src/main/resources/static/images/landingpage-logo.png) (ensure you keep the same file name)
```

Comment:
It looks like someone added the text "customer logo" to this image. This image should just be the logo without any text. This applies to all instances of images like this throughout the repo. Also I think we can leave this instruction step out, we don't get into branding changes for any of these READMEs.

Reply:
All image related changes will be addressed in release 2.0.3.
Issues #3181, #3182, #3183, #3184 and #3185, to update the images for all modules.